### ISSUE TITLE
[Proposal] Убрать все внешние признаки того что человека убил генокрад

### ISSUE THREAD

**Issue by simb11** [TauCetiStation/TauCetiClassic#10054]
#### Подробное описание проблемы
 Убрать все внешние признаки того что человека убил генокрад, убрать отсутствие крови, сломанную шею и описание в экзамайне. Сделать так, чтобы определить то, что человека убил генокрад можно было только через вскрытие. Спрашивается, зачем нах почему???? Ответ прост: чтоб нельзя было спалить режим по одному лишь экзмайну трупика.

**Comment by kalazus** [comment]
Так он и не определяется. Если я правильно помню, то труп выглядит как после долгого нахождения в космосе или долго горящий.

**Comment by simb11** [comment]
Там при осмотре видно, типа, в башке трупа огромная дырень.

**Comment by Deahaka** [comment]
Ну да, экзамайн трупа от генокрада с 99% означает что режим генки. Но с другой стороны а что должно быть иначе? Это ж высосанное тело ёмаё. Тут просто генокрад уже не тайна и получается что сразу известно что с жертвой случилось.

**Comment by Mercurialaste** [comment]
Отрезать голову так сложно?

**Comment by simb11** [comment]
> Отрезать голову так сложно?

да, норм оружие, которым можно отрезать голову есть всего у пары проф.

**Comment by DarthSidiousPalpatine** [comment]
> > Отрезать голову так сложно?
> 
> да, норм оружие, которым можно отрезать голову есть всего у пары проф.

кусачки?

**Comment by Mercurialaste** [comment]
> > > Отрезать голову так сложно?
> > 
> > 
> > да, норм оружие, которым можно отрезать голову есть всего у пары проф.
> 
> кусачки?

Ломик???

**Comment by simb11** [comment]
там уже как сто лет нельзя отрезать головы всякими кусачками и осколками стекла

**Comment by flying-george** [comment]
Рукой-мечем отрезаешь просто и все... Или в третьем грабе. Или тело в космос выкинуть. Или просто разварить и затыкать. Или пустит на мясо у повара. Вариантов миллион, а самый простой вскрыть голову мечом

**Comment by Deahaka** [comment]
Рука-меч не острая, если что XDDD

**Comment by artur0116** [comment]
> Рука-меч не острая, если что XDDD

ну так отрубишь голову,а труп серого цвета останется все равно.Если кто то увидет труп без головы в ящике без одежды он подумает такой:"Бля, точно не генокрады режим,слаги брать не буду,у него же головы нет!Как через экзайм теперь определить то..."Люди же не глупые в конце концов

**Comment by artur0116** [comment]
> > > Отрезать голову так сложно?
> > 
> > 
> > да, норм оружие, которым можно отрезать голову есть всего у пары проф.
> 
> кусачки?

#8396

**Comment by TauKitty** [comment]
Эта проблема была упомянута в **Tau Ceti Station Forum**. Там могут быть соответствующие подробности:

https://forum.taucetistation.org/t/obsuzhdenie-golosovanie-apdejt-genokradov/36786/1

**Comment by TauKitty** [comment]
Эта проблема была упомянута в **Tau Ceti Station Forum**. Там могут быть соответствующие подробности:

https://forum.taucetistation.org/t/a-chto-zhe-s-genokradami/40609/25
